.eslintrc.yaml

```diff
@@ -0,0 +1,7 @@
+---
+    parser: babel-eslint
+    plugins: 
+        - eslint-plugin-import
+    rules: 
+        no-extraneous-dependencies: 0
```

Comment:
This should only apply to non-released code.

Reply:
Removed it: https://github.com/cerner/terra-graphs/pull/5/commits/179712ae96ba0695fb5569c1615049606f80d2cf